lib/loader.js

```diff
@@ -180,7 +180,7 @@ function createLoader() {
                 var modPath = path.resolve(dir, file);
                 var mod;
                 try {
-                    mod = require(modPath);
+                    mod = subRequire(modPath);
```

Comment:
Middleware mocks were not loading because middleware was not loaded with `subRequire`. I did this to make it work, but admittedly do not understand all of the potential ramifications of this change.